src/resources/dog/controller.js

```diff
@@ -46,21 +46,21 @@ const getOneById = async (req, res) => {
 
 async function createOneDogAndParticipant(req, res) {
   console.log({ body: req.body });
-  const { name, age, breed, img, shotStatus } = req.body;
+  const { name, petAge, breed, img, shotStatus } = req.body;
   const { firstName, lastName, vaccinated } = req.body.participant;
   try {
     const result = await prisma.dog.create({
       data: {
         name,
-        age,
+        age: parseInt(petAge),
```

Comment:
You've changed the schema from `age` to `petAge`, so you'll have to update the property here as well